internal/config/config_test.go

```diff
@@ -49,6 +49,12 @@ func TestGetConfig(t *testing.T) {
 	assert.Equal(t, 30*time.Second, result.DataPlaneConfig.Nginx.ReloadMonitoringPeriod)
 	assert.False(t, result.DataPlaneConfig.Nginx.TreatWarningsAsError)
 
+	require.NotNil(t, result.Metrics)
+	assert.True(t, result.Metrics.Collector)
+	assert.Equal(t, "localhost:3000", result.Metrics.OTLPExportURL)
+	assert.Equal(t, "localhost:4318", result.Metrics.OTLPReceiverURL)
+	assert.Equal(t, "/var/etc/nginx-agent/nginx-agent-otelcol.yaml", result.Metrics.CollectorConfigPath)
```

Comment:
this path looks unusual, even for a test

Reply:
Heh, I can change it! Didn't really put a lot of thought into the path considering it's a unit test 😅